Enigmatry.Entry.Blueprint.Domain/Identity/UserContext.cs

```diff
@@ -0,0 +1,7 @@
+namespace Enigmatry.Entry.Blueprint.Domain.Identity;
+
+public class UserContext(Guid id, PermissionsContext permissions)
+{
+    public Guid Id => id;
```

Comment:
Shouldn't it be UserId? Or is that too obvious?

Reply:
I will rename it to UserId, it makes sense.